components/automate-chef-io/content/docs/saml.md

```diff
@@ -82,6 +83,14 @@ The `groups_attr` setting is optional. However, if it is not provided,
 users authenticating via SAML will not be members of any teams.
 {{% /warning %}}
 
+When `allowed_groups` is set, only users with at least one of the configured
+groups are allowed to sign into Chef Automate using SAML.
+Furthermore, all of the user's groups that are _not_ in the list are discarded,
+and not available to Chef Automate.
```

Comment:
I am not seeing how this is accomplished in your code changes, or is it just a fact of the SAML handler that we are documenting here...?

Reply:
The relevant code change was upstream, we make use of it by changing our config.

Reply:
The relevant code changes upstream are https://github.com/dexidp/dex/pull/1544 and https://github.com/dexidp/dex/pull/1704